Essentials/src/com/earth2me/essentials/commands/Commandtpa.java

```diff
@@ -35,13 +35,24 @@ public void run(Server server, User user, String commandLabel, String[] args) th
         }
         // Don't let sender request teleport twice to the same player.
         if (user.getConfigUUID().equals(player.getTeleportRequest()) && player.hasOutstandingTeleportRequest() // Check timeout
-            && player.isTpRequestHere() == false) { // Make sure the last teleport request was actually tpa and not tpahere
+            && !player.isTpRequestHere()) { // Make sure the last teleport request was actually tpa and not tpahere
             throw new Exception(tl("requestSentAlready", player.getDisplayName()));
         }
+
+        final Teleport teleport = player.getTeleport();
```

Comment:
I moved this over here so we only need to use `player.getTeleport()` once (and not again later on in the program).